src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  let totalCost = cost * days;
+
+  if (days >= 7) {
+    return (totalCost -= 50);
+  }
+
+  if (days >= 3) {
+    return (totalCost -= 20);
+  }
+
+  return totalCost;
```

Comment:
The function name 'calculateRentalCost' is clear and describes its purpose well.